packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-labels.spec.ts

```diff
@@ -8,6 +8,14 @@ import {
 } from '../../label-provider/rich-text-editor';
 import { RichTextEditorPageObject } from '../testing/rich-text-editor.pageobject';
 import { ToolbarButton } from '../testing/types';
+import { getSpecTypeByNamedList } from '../../utilities/tests/parameterized';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+
+type LabelProvider =
+    | 'toggleBold'
+    | 'toggleItalics'
+    | 'toggleBulletList'
+    | 'toggleNumberedList';
```

Comment:
This should also go into `../testing.types.ts`?

Reply:
Updated